qmlui/qlcplus_de_DE.ts

```diff
@@ -836,614 +836,614 @@ Ungesicherte Änderungen gehen verloren, wenn du sie nicht speicherst.</translat
+        <translation>Oberer Wert</translation>
     </message>
 </context>
 <context>
     <name>ExternalControls</name>
     <message>
         <location filename="qml/ExternalControls.qml" line="60"/>
         <source>Add an external controller input</source>
-        <translation type="unfinished"></translation>
+        <translation>Externen Eingang hinzufügen</translation>
     </message>
     <message>
         <location filename="qml/ExternalControls.qml" line="81"/>
         <source>Add a keyboard combination</source>
-        <translation type="unfinished"></translation>
+        <translation>Tastenkombination hinzufügen</translation>
     </message>
 </context>
 <context>
     <name>FixtureChannelDelegate</name>
     <message>
         <location filename="qml/fixturesfunctions/FixtureChannelDelegate.qml" line="151"/>
         <source>Auto (HTP)</source>
-        <translation type="unfinished"></translation>
+        <translation>Auto (HTP)</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureChannelDelegate.qml" line="152"/>
         <source>Auto (LTP)</source>
-        <translation type="unfinished"></translation>
+        <translation>Auto (LTP)</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureChannelDelegate.qml" line="153"/>
         <source>Forced HTP</source>
-        <translation type="unfinished"></translation>
+        <translation>HTP erzwingen</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureChannelDelegate.qml" line="154"/>
         <source>Forced LTP</source>
-        <translation type="unfinished"></translation>
+        <translation>LTP erzwingen</translation>
     </message>
 </context>
 <context>
     <name>FixtureGroupEditor</name>
     <message>
         <location filename="qml/fixturesfunctions/FixtureGroupEditor.qml" line="71"/>
         <source>Group size</source>
-        <translation type="unfinished"></translation>
+        <translation>Gruppengröße</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureGroupEditor.qml" line="101"/>
         <source>Transform the selected items</source>
-        <translation type="unfinished"></translation>
+        <translation>Ausgewählte Elemente transformieren</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureGroupEditor.qml" line="108"/>
         <source>Reset the entire group</source>
-        <translation type="unfinished"></translation>
+        <translation>Gesamte Gruppe zurücksetzen</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureGroupEditor.qml" line="131"/>
         <source>Rotate 90° clockwise</source>
-        <translation type="unfinished"></translation>
+        <translation>Um 90° drehen (im UZS)</translation>
```

Comment:
In most desktop applications, it's "Um 90° nach rechts drehen", which sounds a bit nicer.